# -*- coding: utf-8 -*-
# Generated by Django 1.10.5 on 2018-05-02 19:44
from __future__ import unicode_literals

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):
    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('waybills', '0072_auto_20180427_0116'),
    ]

    operations = [
        migrations.CreateModel(
            name='ExceptionRecord',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('create_dt', models.DateTimeField(auto_now_add=True, verbose_name='\u751f\u6210\u65f6\u95f4')),
                ('remark',
                 models.CharField(blank=True, default='', max_length=500, null=True, verbose_name='\u5907\u6ce8')),
                ('area',
                 models.CharField(blank=True, default='', max_length=100, null=True, verbose_name='\u533a\u57df')),
                ('value', models.DecimalField(decimal_places=2, default=0, max_digits=9, verbose_name='\u4ef7\u503c')),
                ('qty', models.PositiveSmallIntegerField(default=1, verbose_name='\u4e2a\u6570')),
                ('description', models.CharField(blank=True, default='', max_length=100, null=True,
                                                 verbose_name='\u7269\u54c1\u540d')),
                ('user', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL,
                                           related_name='exception_records', to=settings.AUTH_USER_MODEL,
                                           verbose_name='\u63d0\u4ea4\u4eba')),
                ('waybill', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL,
                                              related_name='exception_records', to='waybills.Waybill',
                                              verbose_name='\u8fd0\u5355')),
            ],
        ),
        migrations.CreateModel(
            name='ExceptionRecordImage',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('image', models.ImageField(blank=True, null=True, upload_to='exception_records/%Y/%m',
                                            verbose_name='\u56fe\u7247')),
                ('record', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='images',
                                             to='waybills.ExceptionRecord', verbose_name='\u95ee\u9898\u8bb0\u5f55')),
            ],
        ),
    ]
